taxi/urls.py

```diff
@@ -16,7 +22,25 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/create",
```

Comment:
Consider adding a trailing slash to the 'manufacturers/create' path for consistency with Django's URL conventions.